src/main/java/com/vaadin/training/bugrap/ui/ReportView.java

```diff
@@ -0,0 +1,113 @@
+package com.vaadin.training.bugrap.ui;
+
+import org.vaadin.bugrap.domain.entities.Report;
+import org.vaadin.bugrap.domain.entities.Reporter;
+
+import com.vaadin.data.Binder;
+import com.vaadin.data.ValidationException;
+import com.vaadin.navigator.View;
+import com.vaadin.navigator.ViewChangeListener.ViewChangeEvent;
+import com.vaadin.training.bugrap.scope.UIScope;
+import com.vaadin.training.bugrap.util.CommentFormat;
+import com.vaadin.ui.Button.ClickListener;
+
+public class ReportView extends ReportViewDesign implements View {
```

Comment:
We don't usually nest views inside views, so perhaps it would be best to remove the "implements View" -part of. Then once you have the separate UI for single report in place then you can add a new view that uses this component as it's content

Reply:
Ok!